setup.py

```diff
@@ -21,7 +21,7 @@ def read(fname):
       long_description=read('README.rst'),
       packages=find_packages(),
       install_requires=['numpy >= 1.7.0, < 1.17',
-                        'pandas >= 0.18.0, < 0.25'],
+                        'pandas >= 0.18.0, < 1.2'],
```

Comment:
```suggestion
                        'pandas >= 0.18.0'],
```
already suggested ;)

Reply:
numpy, too?

Reply:
But then we should check whether the main function work with the newest numpy versions, too..